### DISCUSSION TITLE
can someone help battle 1, the button

### DISCUSSION THREAD

**Discussion by obeverelst** [cssbattle/forum#280] [General]
so I tried Battle #1 'Pilot Battle', The Button, it looks like it's a 100% match but it isn't, I'll share my code, and can someone maybe tell me why it isn't a 100%, also I'm new to CSS Battle. I had it the first time but tried it again because I wanted to make it shorter. Now I get a 98.2% math but I can't find any difference

code:
```
p id=r
p id=c

<style>
  
  body {
    background:#6592CF;
  }
  
  p {
    position:absolute;
    margin:0
  }
  
  #r {
    width:300;
    height:150;
    background:#243D83;
    top:75;
    left:50
  }
  
  #c {
    width:50;
    height:50;
    border-radius:50%;
    background:#EEB850;
    top:125;
    left:175;
    box-shadow:0 0 0 50px #243D83, 0 0 0 100px #6592CF
  }
```

**Comment by chinchang** [comment]
@obeverelst Make sure you are on latest Chrome (or other Chromium based browser). Because this is how your code renders on latest Chrome:

![image](https://user-images.githubusercontent.com/379918/180149076-bb3ab880-c209-487e-8ed9-572d78932d72.png)
